lte/gateway/c/session_manager/SessionCredit.cpp

```diff
@@ -203,7 +226,6 @@ SessionCredit::Usage SessionCredit::get_usage_for_reporting(
   auto usage = get_unreported_usage();
   // Apply reporting limits since the user is not getting terminated.
   // We never want to report more than the amount we've received
-  apply_reporting_limits(usage);
```

Comment:
If we are getting rid of the reporting limit, should we remove the comments here then? :D  Also just curious, did you ever figure out why we put the logic in there in the first place?